tests/Benchmark/SpanBench.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Tests\Benchmark;
+
+use PhpBench\Benchmark\Metadata\Annotations\Iterations;
+use PhpBench\Benchmark\Metadata\Annotations\Revs;
+use Sentry\Tracing\Span;
+use Sentry\Tracing\TransactionContext;
+
+class SpanBench
+{
+    /**
+     * @var TransactionContext
+     */
+    private $context;
+    /**
+     * @var TransactionContext
+     */
+    private $contextWithTimestamp;
+
+    public function __construct()
+    {
+        $this->context = TransactionContext::fromSentryTrace('566e3688a61d4bc888951642d6f14a19-566e3688a61d4bc8-0');
+        $this->contextWithTimestamp = TransactionContext::fromSentryTrace('566e3688a61d4bc888951642d6f14a19-566e3688a61d4bc8-0');
+        $this->contextWithTimestamp->setStartTimestamp(microtime(true));
+    }
+
+    /**
+     * @Revs(100000)
+     * @Iterations(10)
+     */
+    public function benchConstructor(): void
+    {
+        $span = new Span();
```

Comment:
We can remove the assignation, as it's useless. Same thing for the other methods 😄

Reply:
I'm not sure we can. It may give space for optimizing out the instantiation of the class, making the bench unuseful.

Reply:
I doubt that there is this kind of optimization as PHP cannot know what the constructor of such class do (it may set some global variables for example). Anyway, if you feel safer, let's keep it as is